### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by kvklink**
Added a documentation page that describes the process of back-filling existing products into the orchestrator.

This is an approach we followed within GÉANT for importing the existing estate, and worked quite well for us. As we had our usual meeting with the fine folks from HEAnet, we decided it would be good to document this for future reference and to minimise wheel-reinvention.

### CODE REVIEW

File: docs/architecture/product_modelling/imports.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+user input from a form page, but could also be provided programmatically by either a CLI command or API endpoint that is
+called. Data sources can include external API resources, CSV- or YAML files, etc. If desired, interaction with all
+external provisioning systems can be skipped, resulting in a creation workflow that could be as simple as follows.
+
+```python
+from orchestrator.workflow import StepList, begin, done, workflow
+from orchestrator.workflows.steps import resync, store_process_subscription
+
+@workflow("Create imported Node", target=Target.CREATE)
+def create_imported_node() -> StepList:
```
Comment (Mark90) [inline docs/architecture/product_modelling/imports.md:37]: Perhaps you already know and intentionally didn't use them, but `orchestrator.workflows.utils` contains wrappers `@create_workflow` and `@modify_workflow` that simplify some of the boilerplate

File: docs/architecture/product_modelling/imports.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        >> unsync
+        >> import_node_subscription
+        >> resync
+        >> done
+    )
+```
+
+In this workflow, the existing `ImportedNode` subscription is modified into a `Node` subscription, and is stored in the
+service database. Now, the import has been completed, and the imported variety of the product has been replaced with a
+"regular" `Node` subscription.
```
Comment (Mark90) [inline docs/architecture/product_modelling/imports.md:88]: Is the ImportedNode subscription actually gone after this workflow?

**Review by Mark90 — APPROVED** [review]
🚀 Thanks!

**Comment by Mark90** [comment]
Perhaps @pboers1988 or @wouter1975 can review this given their experience from the SN7/SN8 migration :)

File: docs/architecture/product_modelling/imports.md (hunk 1 above)
Comment (kvklink) [reply docs/architecture/product_modelling/imports.md:37]: I've seen them, yes, but actually forgot to use them. I'd like to do that, even if it's just as an exercise to see whether I can use them correctly. 🙂

File: docs/architecture/product_modelling/imports.md (hunk 2 above)
Comment (kvklink) [reply docs/architecture/product_modelling/imports.md:88]: Fortunately, yes

**Review by pboers1988 — APPROVED** [review]

**Pull request merged**